example/src/services/PlaybackService.ts

```diff
@@ -1,4 +1,5 @@
 import TrackPlayer, {Event, State} from 'react-native-track-player';
+import type {ProgressUpdateEvent} from 'react-native-track-player';
```

Comment:
It's being referenced as `import {PlaybackService} from './src/services';`, and since the `PlaybackService` is exported from `./src/services/index.ts` this should be correct (unless I'm missing something).